client/landing/gutenboarding/components/domain-picker-modal/style.scss

```diff
@@ -0,0 +1,51 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../../mixins.scss';
+
+.domain-picker-modal-overlay {
+	// Absolute positioning allows the modal
+	// to reuse the <body> element's scrollbar.
+	position: absolute;
+
+	// This positions the domain picker modal
+	// right below the gutenboarding header,
+	// keeping the header clickable.
+	top: 64px;
+	left: 0;
+
+	// Using min-height lets the overlay cover
+	// the entire viewport ensuring nothing behind
+	// it can be seen.
+	//
+	// When the domain picker's content is taller
+	// than the viewport height, it will expand taller
+	// than the provided min-height, triggering
+	// the appearance of the <body> element's scrollbar.
+	min-height: calc( 100vh - 64px );
+	width: 100%;
+
+	background: var( --studio-white );
+}
+
+.domain-picker-modal {
+
+	.domain-picker__panel-row-main {
+		padding: 64px 88px;
```

Comment:
These numbers seem a bit magical. Can comment on what they are and maybe use variables.

Reply:
Thanks! They are intended to mimic onboarding block's margins and header margins. All ambigous numbers have been replaced with variables or proper mixins. https://github.com/Automattic/wp-calypso/pull/41212/commits/e608b09df835c8d0c1a40e6568c2789b923f2ed2